.travis.yml

```
@@ -0,0 +1,5 @@
+language: node_js
+node_js: 'node'
+
+script:
+  - npm run all
```

We're going to need this for test coverage reports:

```suggestion
  - COVERAGE=1 npm run all
  - npx codecov
```